diagnostic.md

```diff
@@ -51,13 +64,21 @@ Write the `curl` request you'd use to delete a single donut, using whatever
 ID you'd like.
 
 ```sh
-# your answer here
+curl "http://www.example.com/donuts/1" \
```

Comment:
The url should go at the end: `curl --include --request DELETE http://www.example.com/donuts/1`